packages/pigeon/platform_tests/test_plugin/windows/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@
 # customers of the plugin.
 cmake_minimum_required(VERSION 3.14)
 
+# Due to "number of sections exceeded object file format limit: compile with /bigobj" error
+if(MSVC)
```

Comment:
This doesn't need to be conditionalized; we don't support other compilers for Windows.

Reply:
Have removed the condition.